### DISCUSSION TITLE
Android Kotlin

### DISCUSSION THREAD

**Discussion by Manohar-mj** [community/community#33368] [Other Feature Feedback, Questions, & Ideas]

**Comment by queenofcorgis** [comment]
Hi @Manohar-mj, we're happy you're here 😀

However, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
